analysis/report_templates/default.html

```diff
@@ -274,7 +281,9 @@ <h5 class="center-align">Pairwise unique coverage</h4>
                         <div class="row">
                             {% for fuzzer in benchmark.fuzzer_names %}
                             <div class="col">
-                                <a class="waves-effect waves-light btn" href="{{ benchmark.get_filestore_name(fuzzer) }}/coverage/reports/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
+                                <p>
+                                    <a class="waves-effect waves-light btn" href="{{ benchmark.get_filestore_name(fuzzer) }}/coverage/reports/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
+                                </p>
```

Comment:
Buttons are most typically used for interacting with the current page, and rarely for links.
Consider using a collection of links instead: https://materializecss.com/collections.html#links
(Perhaps even with an "external link" icon: https://fontawesome.com/icons/external-link-alt)

Reply:
The collection list would be too long in a full experiment. Does it matter?

Reply:
Since we put them in a collapsibles, i think it's fine, no?

Reply:
Yeah that makes sense. I'll change it.